example/web/public/.well-known/assetlinks.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "relation": ["delegate_permission/common.handle_all_urls"],
+    "target": {
+      "namespace": "android_app",
+      "package_name": "com.splendo.kaluga.example",
+      "sha256_cert_fingerprints": [
+          "BF:0D:B5:2E:8E:4F:9C:73:94:C7:D8:B9:27:CB:CE:AD:DA:85:45:2D:4A:E5:BF:E0:F7:17:00:18:B9:92:EA:E6"
```

Comment:
same here: package name is fine, but fingerprint has little value for external people

Reply:
I assume the debug key associated with this will also be in the repo (I don't see it though)

Reply:
If I upload the .keystore should I also upload: `alias`, `storePassword` and `keyPassword`? Gradle should include the part where it signs the apk and we need those credentials

Reply:
it's fine for a debug key

Reply:
Then should I share them in the README (and explain to put them in local.properties) or just hardcode them in gradle?

Reply:
41190b3
>I assume the debug key associated with this will also be in the repo (I don't see it though)

Reply:
It's ok to share the credentials of a debug key, to make this example works out of the box. (imho)
I don't see a problem with that and we could use simple values to point out that are not big "secrets"

Although, if you really want to go the local.properties way, here is one way to do it (that CI can deal with):
in `android:build.gradle.kts`:
```
fun getFlexibleProperty(name: String): String {
    return if (project.hasProperty(name))
        project.property(name).toString()
    else {
        gradleLocalProperties(projectDir).getProperty(name) ?:
        gradleLocalProperties(rootDir).getProperty(name) ?:
        "No $name value found in local.properties nor as gradle command line argument"
    }
}

android {
 
    val debugKeyPw: String = getFlexibleProperty("debugKeyPw")
    val debugKeyStorePw: String = getFlexibleProperty("debugKeyStorePw")

    ...

    signingConfigs {
        getByName("debug") {

            keyAlias = "KalugaDebug"
            keyPassword = debugKeyPw
            storeFile = file("$rootDir/android/kaluga-debug.keystore")
            storePassword = debugKeyStorePw
        }
    }
    ...
}

```
Would be nice to have @thoutbeckers  opinion on this, I think it count the most for these kind of things related to Splendo.

Reply:
I don't think it's needed to let users use their own key, example is not meant as a starting point for your own app, it's to demonstrate the workings of the API. If you need to just use our key that is not problematic.